drivers/framework.go

```diff
@@ -14,6 +14,7 @@ type WebhookDriver interface {
 	Execute(config interface{}, apiClient client.RancherClient) (int, error)
 	GetSchema() interface{}
 	ConvertToConfigAndSetOnWebhook(conf interface{}, webhook *model.Webhook) error
+	DefaultValues() map[string]interface{}
 }
```

Comment:
If it's okay to add this method to the interface, then there's no hardcoding required here https://github.com/rancher/webhook-service/blob/master/service/routes.go#L78, else the other way I can think of is adding this block of code
```
if key == "scaleService" {
	if k == "min" {
		f.Default = 1
	}
	if k == "max" {
		f.Default = 100
	}
}
```
over here https://github.com/rancher/webhook-service/blob/master/service/routes.go#L98